.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,15 @@
-<!--
-****Please describe the pull request motivation and changes here
-along with non-obvious things.
--->
+## Overview
+<!-- Please describe your changes here and list any open questions you might have. -->
+
+---
+<!-- This is for Exonum Team members only. -->
+See: https://jira.bf.local/browse/ECR-XYZW
+
+### Definition of Done
+
+- [ ] There are no TODOs left in the code
```

Comment:
```suggestion
- [ ] There are no TODOs left in the merged code
```

Reply:
Not that important actually :)

Reply:
Yep, I think it's pretty obvious though :)